modules/gitops-configuring-high-availability-for-argo-rollouts.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/enabling-ha-support-for-argo-rollouts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-configuring-high-availability-for-argo-rollouts_{context}"]
+= Configuring high availability for Argo Rollouts
+
+To enable high availability, configure the `ha` specificaiton in the `RolloutManager` CR by completing the following steps:
```

Comment:
```suggestion
To enable high availability, configure the `ha` specification in the `RolloutManager` CR by completing the following steps:
```
typo :)

Reply:
Ugghhh, sorry about the typo. It shouldn't have been there. Maybe I need to find a spellchecker extension for my MVS editor. Fixed.